runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingModeExecutionContext.java

```diff
@@ -292,41 +213,54 @@ protected SideInputReader getSideInputReaderForViews(
    * Fetches the requested sideInput, and maintains a view of the cache that doesn't remove items
    * until the active work item is finished.
    *
-   * <p>If the side input was not ready, throws {@code IllegalStateException} if the state is
-   * {@literal CACHED_IN_WORKITEM} or returns null otherwise.
-   *
-   * <p>If the side input was ready and null, returns {@literal Optional.absent()}. If the side
-   * input was ready and non-null returns {@literal Optional.present(...)}.
+   * <p>If the side input cached, throws {@code IllegalStateException} if the state is {@literal
```

Comment:
should this be: If the side input was *not* cached?

Reply:
done.